<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnkiMorph Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="note_filters_tab">
        <attribute name="title">
         <string>Note Filters</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="note_filters_table">
             <column>
              <property name="text">
               <string>Note Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Tags</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Field</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Morphemizer</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Morph Priority</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Read</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Modify</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="deleteRowPushButton">
               <property name="text">
                <string>Delete Selected Row</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addNewRowPushButton">
               <property name="text">
                <string>Add New Row</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="extra_fields_tab">
        <attribute name="title">
         <string>Extra Fields</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">margin-bottom: 10px;</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;AnkiMorphs can automatically add the following fields to your cards and populate them.&lt;/p&gt;&lt;p&gt;Read the &lt;a href=&quot;https://mortii.github.io/anki-morphs/user_guide/setup/settings/extra-fields.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8783ff;&quot;&gt;guide&lt;/span&gt;&lt;/a&gt; for more info.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="textFormat">
              <enum>Qt::MarkdownText</enum>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="extraUnknownsCheckBox">
             <property name="text">
              <string>am-unknowns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="extraUnknownsCountCheckBox">
             <property name="text">
              <string>am-unknowns-count</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="extraHighlightedCheckBox">
             <property name="text">
              <string>am-highlighted</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="extraDifficultyCheckBox">
             <property name="text">
              <string>am-difficulty</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="restoreExtraFieldsPushButton">
               <property name="text">
                <string>Restore Default Extra Fields Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tags_tab">
        <attribute name="title">
         <string>Tags</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>One unknown morph</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Multiple unknown morphs</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Learn card now</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Set known and skip</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>All morphs known</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLineEdit" name="tagReadyLineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="tagNotReadyLineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="tagLearnCardNowLineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="tagKnownManuallyLineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="tagKnownAutomaticallyLineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="restoreTagsPushButton">
               <property name="text">
                <string>Restore Default Tags Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="preprocess_tab">
        <attribute name="title">
         <string>Preprocess</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QCheckBox" name="preprocessIgnoreSquareCheckBox">
             <property name="text">
              <string>ignore content in square brackets []</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="preprocessIgnoreRoundCheckBox">
             <property name="text">
              <string>ignore content in round brackets（）</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="preprocessIgnoreSlimCheckBox">
             <property name="text">
              <string>ignore content in slim round brackets ()</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="preprocessIgnoreSuspendedCheckBox">
             <property name="text">
              <string>Ignore content in suspended cards</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="preprocessIgnoreNamesMizerCheckBox">
             <property name="text">
              <string>Ignore names found by the morphemizer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="preprocessIgnoreNamesFileCheckBox">
             <property name="text">
              <string>Ignore names found in names.txt</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="restorePreprocessPushButton">
             <property name="text">
              <string>Restore Default Preprocess Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="skip_tab">
        <attribute name="title">
         <string>Skip</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QCheckBox" name="skipKnownCheckBox">
             <property name="text">
              <string>Skip cards with only known morphs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="skipAlreadySeenCheckBox">
             <property name="text">
              <string>Skip cards that have unknown morphs already seen today</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="skipNotificationsCheckBox">
             <property name="text">
              <string>Show &quot;skipped x cards&quot; notifications</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QPushButton" name="restoreSkipPushButton">
             <property name="text">
              <string>Restore Default Skip Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="recalc_tab">
        <attribute name="title">
         <string>Recalc</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_21">
             <item>
              <widget class="QCheckBox" name="recalcBeforeSyncCheckBox">
               <property name="text">
                <string>Automatically Recalc before Anki sync</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="recalcSuspendKnownCheckBox">
               <property name="text">
                <string>Suspend new cards with only known morphs</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Min. interval for known morphs:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="recalcIntervalSpinBox"/>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>days</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>37</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="restoreRecalcPushButton">
               <property name="text">
                <string>Restore Default Recalc Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="shortcuts_tab">
        <attribute name="title">
         <string>Shortcuts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Run recalc</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Open settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Browse ready cards with same unknown morph</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Browse all cards with same unknown morph</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Set card as known and skip</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Learn card now</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>View card morphemes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Frequency File Generator</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Readability Report Generator</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QKeySequenceEdit" name="shortcutRecalcKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutSettingsKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutBrowseReadyKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutBrowseAllKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutKnownAndSkipKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutLearnNowKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutViewMorphsKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutFrequencyFileGeneratorKeySequenceEdit"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcutReadabilityReportGeneratorKeySequenceEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="restoreShortcutsPushButton">
             <property name="text">
              <string>Restore Default Shortcut Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="restoreAllDefaultsPushButton">
         <property name="text">
          <string>Restore All Default Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="ankimorphs_version_label">
         <property name="text">
          <string>AnkiMorphs version: x</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelPushButton">
         <property name="text">
          <string> Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePushButton">
         <property name="text">
          <string> Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
